### DISCUSSION TITLE
NavbarPage in R Shiny

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#626] [Q&A] [answered]
<img width="676" alt="image" src="https://user-images.githubusercontent.com/106010284/231536413-6a65949b-aaa8-471f-bab3-7015d8f05559.png">

Saya memiliki konten yang banyak, sampai harus discroll ke bawah. Namun saat discroll, navbarPagenya jadi tidak terlihat. Bagaimana caranya agar navbarPagenya tetap terlihat walaupun kontennya discroll ?

Kode saya seperti ini : 
<img width="512" alt="image" src="https://user-images.githubusercontent.com/106010284/231537057-4e80bc24-51a4-432c-bcf0-c5528913b538.png">

Mohon bantuannya kak, terima kasih 🙏

**Comment by amaliasf** [comment, marked as answer]
Dear Pak @yohanespm77, agar navbar tetap terlihat walaupun di scroll Bapak dapat menambahkan parameter `position = "fixed-top"` pada `navbarPage()`.

Berikut referensi yang bisa ditinjau: [Keep NavBar at Top of Screen](https://stackoverflow.com/questions/44661240/r-shiny-navbarpage-keep-navigation-bar-at-top-of-screen)

Silahkan boleh dicoba terlebih dahulu Pak, dan diinfokan kalau masih terkendala.

**Reply by yohanespm77** [reply]
Saya nambahin paramater position = "fixed-top", namun hasilnya jadi begini kak : 

<img width="680" alt="image" src="https://user-images.githubusercontent.com/106010284/231643198-8dcb8ef2-ce7a-48d3-9911-0b1699a06a05.png">

Contentnya terpotong.
